Services/CO.CDP.Organisation.WebApi.Tests/UseCase/InvitePersonToOrganisationUseCaseTest.cs

```diff
@@ -0,0 +1,94 @@
+using CO.CDP.GovUKNotify;
+using CO.CDP.Organisation.WebApi.Model;
+using CO.CDP.Organisation.WebApi.UseCase;
+using CO.CDP.OrganisationInformation.Persistence;
+using GovukNotify.Models;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Persistence = CO.CDP.OrganisationInformation.Persistence;
+
+namespace CO.CDP.Organisation.WebApi.Tests.UseCase;
+
+public class InvitePersonToOrganisationUseCaseTest
+{
+    private readonly Mock<Persistence.IOrganisationRepository> _organisationRepository = new();
+    private readonly Mock<Persistence.IPersonInviteRepository> _personsInviteRepository = new();
+    private readonly Mock<IGovUKNotifyApiClient> _mockGovUKNotifyApiClient = new();
+    private readonly IConfiguration _mockConfiguration;    
+    private readonly Guid _generatedGuid = Guid.NewGuid();
+
+    private InvitePersonToOrganisationUseCase _useCase => new(
+        _organisationRepository.Object,
+        _personsInviteRepository.Object,
+        _mockGovUKNotifyApiClient.Object,
+        _mockConfiguration,
+        () => _generatedGuid);
+
+    public InvitePersonToOrganisationUseCaseTest()
+    {
+        var inMemorySettings = new Dictionary<string, string> {
+            {"GOVUKNotify:PersonInviteEmailTemplateId", "test-template-id"}
+        };
+
+        _mockConfiguration = new ConfigurationBuilder()
+            .AddInMemoryCollection(inMemorySettings)
```

Comment:
The expected argument type for `AddInMemoryCollection` is `IEnumerable<KeyValuePair<string, string?>>`.

Reply:
Done